src/cdk/lambda/appLayer.ts

```diff
@@ -57,19 +57,37 @@ export class AppLayer extends LayerVersion {
     // node_modules output (in docker)
     const nm = "/asset-output/nodejs/node_modules"
 
+    const prismaDirs = ["prisma", ".prisma", "@prisma", "prisma-appsync"]
+
     // exclude from copying to build environment to speed things up
-    const exclude: string[] = ["*", "!node_modules/.prisma", "!node_modules/@prisma", "!node_modules/prisma"]
+    const exclude: string[] = ["*", "!node_modules/.bin", ...prismaDirs.map((d) => `!node_modules/${d}`)]
```

Comment:
I think I've seen these changes in some other PR

Reply:
yeah the later one was approved and merged